src/app/gps/qgsgpsinformationwidget.h

```diff
@@ -67,10 +72,13 @@ class QgsGPSInformationWidget: public QWidget, private Ui::QgsGPSInformationWidg
     void on_mBtnResetFeature_clicked();
 // not needed    void on_mCbxAutoAddVertices_toggled( bool flag );
     void on_mBtnLogFile_clicked();
-
     void connected( QgsGPSConnection * );
     void timedout();
-
+    void switchAcquisition();
+    void on_cboAcquisitionIntervalActivated( const QString & );
```

Comment:
Please remove the "on_" prefix for these slots - otherwise they look like autoconnected slots (which are bad)